test/k8sT/Updates.go

```diff
@@ -89,12 +90,13 @@ var _ = Describe("K8sUpdates", func() {
 		ExpectAllPodsTerminated(kubectl)
 	})
 
-	It("Tests upgrade and downgrade from a Cilium stable image to master", func() {
-		var assertUpgradeSuccessful func()
-		assertUpgradeSuccessful, cleanupCallback =
-			InstallAndValidateCiliumUpgrades(kubectl, helpers.CiliumStableVersion, helpers.CiliumDevImage())
-		assertUpgradeSuccessful()
-	})
+	SkipItIf(func() bool { return os.Getenv("NO_CILIUM_ON_NODE") != "" },
+		"Tests upgrade and downgrade from a Cilium stable image to master", func() {
+			var assertUpgradeSuccessful func()
+			assertUpgradeSuccessful, cleanupCallback =
+				InstallAndValidateCiliumUpgrades(kubectl, helpers.CiliumStableVersion, helpers.CiliumDevImage())
+			assertUpgradeSuccessful()
+		})
 })
```

Comment:
Then change the files under `test/k8sT/manifests/v1.5` to accommodate such annotations

Reply:
Upgrade from v1.5 to the latest when running without kube-proxy doesn't make sense, as we introduced the kube-proxy replacement only in v1.6.

Reply:
@brb that's not what the code is doing, what is doing is checking if `os.Getenv("NO_CILIUM_ON_NODE") != ""`, there's no `kube-proxy` mentioned nor even with a comment.

Reply:
Updated.